llvm/test/Transforms/InterleavedAccess/AArch64/sve-interleaved-accesses.ll

```diff
@@ -491,6 +491,30 @@ define void @store_bfloat_factor2(ptr %ptr, <16 x bfloat> %v0, <16 x bfloat> %v1
   ret void
 }
 
+; Use 4 vliad llvm.vector.insert.nxv4f64.nxv2f64
```

Comment:
This is effectively a negative test so perhaps something like "Ensure vscale_range property does not affect scalable vector types."?

Reply:
Update the comment, thanks